spec/cucumber/formatter/http_io_spec.rb

```diff
@@ -80,16 +80,16 @@ module Formatter
       context 'created by Io#ensure_io' do
         it 'returns a IOHTTPBuffer' do
           url = start_server
-          io = ensure_io("#{url}/ -X POST")
+          io = ensure_io("#{url}/s3 -X POST")
```

Comment:
that one is not that obvious because WebRick does not support `PUT` by default. I won't be able to finish it for today https://stackoverflow.com/questions/4996170/handle-the-put-method-in-webrick

Reply:
Try adding the `webrick` gem, it might be more recent than the one supplied in the stdlib, and might support it.

Reply:
webrick gem was already used. I have aliased `do_PUT` to `do_GET` in webrick internal classes to get it working.